packages/react-native-gradle-plugin/build.gradle.kts

```diff
@@ -8,7 +8,7 @@
 plugins {
   `java-gradle-plugin`
   `kotlin-dsl`
-  kotlin("jvm") version "1.4.21"
```

Comment:
This was a deliberate downgrade?

I don't have any specific knowledge, just noticed it was the only number going down not up! 😅

Reply:
it was printing warning, so removed it 😅